HackingWithSwift/SwiftUI/myProject2/myProject2/myProject2/Emoji.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Emoji.swift
+//  myProject2
+//
+//  Created by Katasee on 20/10/2024.
+//
+import SwiftUI
+
+
+struct Emoji: View {
```

Comment:
I suggest using more descriptive names. For example, EmojiImage would be a better fit here as it clearly describes the purpose.